mkinit/__main__.py

```diff
@@ -174,9 +185,10 @@ def main():
         "with_attrs": ns["with_attrs"],
         "with_mods": ns["with_mods"],
         "with_all": ns["with_all"],
-        "relative": ns["relative"],
+        "relative": ns["relative"] or ns["lazy_loader_typed"],

```

Comment:
Note that the `attach_stub` method requires relative paths, so I always have relative as true for `lazy_loader_typed`

Reply:
I don't quite understand why `attach_stub` requires relative paths. While I would like it to work with absolute imports (which I tend to prefer because they copy / paste much nicer), it's out of scope for this repo. In any case, I think I would rather either:

1. raise an error if incompatable settings are given
2. fallback to the relative behavior but emit a warning to let the user know we are doing that.

Reply:
Sounds good, I'll opt for the second option, as it feels wrong to _require_ the use of two args to make the mode work, whereas a warning still keeps everything transparent.

This looks like the relevant issue regarding why relative is needed: https://github.com/scientific-python/lazy_loader/issues/52. It does seem to be arbitrary, but it's not an issue I have time to get into.